dbt_project.yml

```diff
@@ -31,9 +31,9 @@ clean-targets:         # directories to be removed by `dbt clean`
 # as tables. These settings can be overridden in the individual model files
 # using the `{{ config(...) }}` macro.
 models:
-    jaffle_shop:
-      materialized: table
-      staging:
-        materialized: view
-        schema: stg
-        tags: "staging"
+    mart:
```

Comment:
The top-level `models` config always needs to be the name of your project, after which it can be folders within that project. See [docs with an example](https://docs.getdbt.com/docs/writing-code-in-dbt/jinja-context/dbt-project-yml-context/#example-configuration).

So you'll need to keep `jaffle_shop` here, and then `mart` indented beneath it, assuming that's still the name of your project.

The reason for doing this is so that you can also configure models in packages, which we'll talk more about tomorrow.